requirements.txt

```diff
@@ -7,4 +7,4 @@ pyyaml
 BeautifulSoup4==4.11.*
 python-keycloak
 prometheus_api_client
-pydantic==1.10.*
```

Comment:
I don't think you expected to remove this, right?
(one commit puts it, the next one removes it)

Reply:
I tried to pin the version via the requirements file, but it didn't actually take hold in the middleware CI..

So I then tried using Pydantic's v1 endpoint (meant for backwards compatibility), but hit a similar issue

Reply:
pinning the requirement file worked (I merged that on Friday), maybe the middleware CI had issues with with the submodule not being properly updated? 🤔 
when in doubt, try in OpenShift CI, it won't have such issues

supporting Pydantic v2, first via `pydantic.v1` then with proper pydantic v2 modules would be a good thing,
but for now I think we're good enough with the pinned package